baseband/vlbi_base/base.py

```diff
@@ -255,10 +258,8 @@ def tell(self, unit=None):
     def _frame_info(self):
         offset = (self.offset +
                   self.header0['frame_nr'] * self.samples_per_frame)
-        framerate = int(np.round(
```

Comment:
I hadn't quite appreciated that framerate was only used here. I would suggest to also only store it here, so that, e.g., `dada` or `gsb` don't get an extraneous private property that is, for them, incorrect.  At the same time we could also avoid the `header0['frame_nr'`]` call (which is only meaningful for VLBI formats). So, something like
```
try:
    framerate = self._framerate
    offset0 = self._offset0
except AttributeError:
    framerate = self._framerate = int(...)
    offset0 = self._offset0 = self.header0['frame_nr'] * self.samples_per_frame

offset = self.offset - offset0
```